tests/ui/concrete-playback/cover/main.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// kani-flags: --enable-unstable --concrete-playback=print
+
+#[kani::proof]
+pub fn harness() {
+    let x: i32 = kani::any();
+    kani::cover!(x != 0 && x / 2 == 0);
```

Comment:
Can you make this check to have only one satisfying value, so we avoid having to change the test whenever CBMC decides to produce a different value.

Reply:
Good point. Done.